### PR TITLE
Ading files and folders

### PR DESCRIPTION
**Pull request by TiteLunda**
Contact form added to the mob version portofolio project

### CODE REVIEW

**Comment by TiteLunda** [comment]
image
I attached this screenshot as proof allowing me to do projects this week solo.

File: index.html [hunk 1]
```diff
@@ -185,6 +185,50 @@ <h2 class="Recent ">About-me</h2>
                 </div>
             </div>
         </section>
+        <section class="form">
+            <h1 class="about-contact">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h1>
+            <form action="">
+                <div class="inputs">
```
Comment (hiromataba) [inline index.html]: - [ ] You should use formspree service to collect data of your form and your form should have a post method 

image

File: index.html [hunk 2]
```diff
@@ -185,6 +185,50 @@ <h2 class="Recent ">About-me</h2>
...
+                            <label for="message"></label>
+                            <textarea type="text" name="message" rows="5" placeholder="Enter yout text here" maxlength="500" class="text" required></textarea>
+                        </li>
+                    </ul>
+
+                </div>
+                <div class="button-contact">
+                    <button class="button">
+                      Get in touch
+                    </button>
```
Comment (hiromataba) [inline index.html]: - [ ]. Your button should have submit as type in order to submit the form

File: index.html [hunk 3]
```diff
@@ -185,6 +185,50 @@ <h2 class="Recent ">About-me</h2>
                 </div>
             </div>
         </section>
+        <section class="form">
+            <h1 class="about-contact">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h1>
```
Comment (hiromataba) [inline index.html]: - [ ] Please check my feedback from [ the video](https://youtu.be/Dmm04Kyzpyg) I recorded for you

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @TiteLunda    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @TiteLunda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Form submission is working fine👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**